concatenate_ice/concat_ice_daily.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+#concatenate sea-ice daily output
+#script inspired from https://github.com/COSIMA/1deg_jra55_ryf/blob/master/sync_data.sh#L87-L108
+
+out_dir=$(ls -td archive/output??? | head -1)/ice/OUTPUT #latest output dir only
+
+for f in $out_dir/iceh.????-??-01.nc; do
+    #concat daily files for this month
+    echo "doing ncrcat -O -L 5 -4 ${f/-01.nc/-??.nc} ${f/-01.nc/-daily.nc}"
+    ncrcat -O -L 5 -4 ${f/-01.nc/-??.nc} ${f/-01.nc/-daily.nc} 
```

Comment:
```suggestion
    {PAYU_PATH}/ncrcat -O -L 5 -4 ${f/-01.nc/-??.nc} ${f/-01.nc/-daily.nc} 
```

Reply:
Yeah but the script still needs to be able to find `ncrcat` ?

Reply:
Oh good point. Yes that is inside the script. Soz. 

Some options are:

1. hard-code path to `ncrcat` in the script (very fragile)
2. require `ncrcat` to be "magically available" when the script is called ([what I suggested](https://github.com/ACCESS-NRI/payu-condaenv/issues/23#issuecomment-2022045732), but also fragile)
3. put `module load nco` in the script ([as you suggested](https://github.com/ACCESS-NRI/payu-condaenv/issues/23#issuecomment-2021842903))

I don't *like* any of them much, but 3 seems the least worse option with the only downside being portability. We should use a specific version that has been tested and known to work to reduce fragility/potential breakage.

Sorry for flip-flopping on this so much.

Reply:
I briefly entertained the idea of putting `module load` back into the script, but it can be kinda horrible if there is already an `nco` module loaded.

If you wanted to be very nice to the user you could [check if `ncrcat` is available](https://unix.stackexchange.com/a/696354) and emit a nice message to suggest possible remedies if it was not. But it will fail and say it can't find `ncrcat` which should be enough information I think.